using_images/db_images/mongodb.adoc

```diff
@@ -134,3 +134,278 @@ The MongoDB image can be run with mounted volumes to enable persistent storage f
 
 * *_/var/lib/mongodb_* - This is the database directory where
 MongoDB stores database files.
+
+=== Using MongoDB replication
+
+IMPORTANT: The replication support provided by the MongoDB image is experimental and
+should not be used in production.
+
+The full source of replication example is available here:
+https://github.com/openshift/mongodb/tree/master/2.4/examples/replica
+
+IMPORTANT: The template provided in the example above does not use persistent
+storage. When you loose all members of the replication set, your data will be
+lost.
+
+This section describes how to start a cluster of MongoDB servers that implements
+master-slave replication and automated failover. This is the MongoDB’s
+recommended replication strategy.
+
+==== Creating the DeploymentConfig
+
+To start with MongoDB replication, you first have to define a DeploymentConfig
+that defines a replication controller which will manage the members of the
+MongoDB cluster. To tell a MongoDB server that the member will be part of the
+cluster, you have to provide additional environment variables for the container
+defined in the replication controller pod template:
```

Comment:
actually I take it back, I guess you're only talking about the member DC here.  I would put the section that describes the one-shot service/pod before this section.